### PR TITLE
Vscode v1.31+

### PR DESCRIPTION
**Pull request by BernyGA**
Modify for VS Code v1.31+
Compile dist for OJET v6.2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+Starting with Visual Studio Code 1.31, there is (at this moment) experimental support for custom elemements/attributes (see [https://code.visualstudio.com/updates/v1_31#_html-and-css-custom-data-support) for details)
 
 This repository contains file `dist/tags.json` that contains OJET components and their attributes:
 1. Download this JSON file to your PC
 2. Open VS Code settings and reference this file in following way:
 ```
-    "html.experimental.custom.tags": [
-        "/path/to/downloaded/file/tags.json"
+	"html.experimental.customData": [
+        "\\path\\to\\downloader\\file\\tags.json"
```
Comment (ladariha) [inline README.md]: Could we leave the paths to be Unix friendly? :) Leave the forward slashes please

File: index.js [hunk 2]
```diff
@@ -58,34 +58,36 @@ function proccessMetadataFile(jsonFile) {
...
-                        a.description = _enumHelp;
-                    }
+                if (component.properties[name].enumValues) {
+                    //const _enumHelp = `[{"name": ${component.properties[name].enumValues.join("}, {\"name\": ")} }]`;
+
+					a.values = component.properties[name].enumValues.map(value => {
+                        const v = {
+                            name: value
+                        }
+                        return v;
```
Comment (ladariha) [inline index.js]: Could you please fix formatting and simple return `{name: value}` instead of initializing a new `v` variable?

**Review by ladariha — CHANGES_REQUESTED** [review]

**Comment by ladariha** [comment]
Thanks a lot for these changes, please resolve the minor issues (formatting, slashes, no extra `v` variable) and I think it is good to be merged.

**Review by BernyGA — COMMENTED** [review]
Made the suggested changes.
If you consider that something is not right you can correct it without problem, since the real work is yours.

**Comment by ladariha** [comment]
Thanks again for your help :)

**Pull request merged**